donate/payments/views.py

```diff
@@ -760,8 +762,16 @@ def get_initial(self):
         return initial
 
     def form_valid(self, form, send_data_to_basket=True):
-        if send_data_to_basket:
-            data = form.cleaned_data.copy()
+        data = form.cleaned_data.copy()
+
+        if hasattr(settings, 'THUNDERBIRD'):
+            newsletter_url = settings.POST_DONATE_NEWSLETTER_URL
+            data = parse.urlencode({'EMAIL': data['email']}).encode()
+            req =  request.Request(newsletter_url, data=data)
+            res = request.urlopen(req)
+            # Do we want Sentry logging for res.status here, in case it's not 200?
```

Comment:
@cadecairos what's the right way to do this, such that the data goes to the folks that need to read it?